### PR TITLE
Iam 1034 o11y improvements

### PR DESCRIPTION
**Pull request by pik4ez-canonical**
Closes https://warthogs.atlassian.net/browse/IAM-1034.

Improving observability:
* Switching to LogForwarder.
* Adding tracing with Tempo.

### CODE REVIEW

File: src/charm.py [hunk 1]
```diff
@@ -131,12 +130,7 @@ def __init__(self, *args: Any):
 
         self.service_patcher = KubernetesServicePatch(self, [("ldap", GLAUTH_LDAP_PORT)])
 
-        self.loki_consumer = LogProxyConsumer(
-            self,
-            log_files=[str(LOG_FILE)],
```
Comment (natalian98) [inline src/charm.py:136]: [This](https://github.com/canonical/glauth-k8s-operator/blob/877a05575d1a8caad4bbd0d5da6f272682592f01/src/charm.py#L260) is not needed anymore

**Review by natalian98 — COMMENTED** [review]
lgtm, but I think we should stop creating the log directory (see the comment below) and simplify the [command](https://github.com/canonical/glauth-k8s-operator/blob/877a05575d1a8caad4bbd0d5da6f272682592f01/src/configs.py#L114-L116) since the logs don't need to be written to the log file anymore

**Review by BarcoMasile — DISMISSED** [review]
LGTM, and I second Natalia's comment

File: src/charm.py (hunk 1 above)
Comment (pik4ez-canonical) [reply src/charm.py:136]: Good points, thank you! Fixed.

**Review by nsklikas — APPROVED** [review]

**Review by natalian98 — APPROVED** [review]

**Pull request merged**